integrations/ad_integration/import_ad_group_into_mo.py

```diff
@@ -0,0 +1,321 @@
+import uuid
+import json
+import pathlib
+import hashlib
+import logging
+import argparse
+import datetime
+
+import requests
+
+from os2mo_helpers.mora_helpers import MoraHelper
+from integrations.ad_integration import payloads
+from integrations.ad_integration.ad_reader import ADParameterReader
+
+# Set up a real logger!
+logger = logging.getLogger("ImportADGroup")
+
+LOG_LEVEL = logging.DEBUG
+LOG_FILE = 'external_ad_users.log'
+
+for name in logging.root.manager.loggerDict:
+    if name in ('ImportADGroup', 'AdReader', 'mora-helper', 'AdCommon'):
+        logging.getLogger(name).setLevel(LOG_LEVEL)
+    else:
+        logging.getLogger(name).setLevel(logging.ERROR)
+
+logging.basicConfig(
```

Comment:
Hvis man skulle finde på at importere det en gang, kunne man lægge denne ned i __main__
Undskyld. Det var bare for at vise at jeg havde læst det. Nu går jeg også hjem.